api/apps/v1alpha1/nemo_evaluator_types.go

```diff
@@ -684,9 +712,15 @@ func (n *NemoEvaluator) GetServiceParams() *rendertypes.ServiceParams {
 	params.Type = "ClusterIP"
```

Comment:
probably not for this PR but a nit:
```suggestion
	params.Type = n.GetServiceType()
```